dev-python/spyder/spyder-5.0.1.ebuild

```
@@ -0,0 +1,202 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8,9} )
```

Could you use the following from (everywhere)?

```suggestion
PYTHON_COMPAT=( python3_{7..9} )
```

Generally we can expect Python to work in complete ranges.